lib/livebook/config.ex

```diff
@@ -316,59 +326,24 @@ defmodule Livebook.Config do
     }
   end
 
-  @doc """
-  Parses file systems list.
-
-  Appends subsequent numbers to the given env prefix (starting from 1)
-  and parses the env variables until `nil` is encountered.
-  """
-  def file_systems!(env_prefix) do
-    Stream.iterate(1, &(&1 + 1))
-    |> Stream.map(fn n ->
-      env = env_prefix <> Integer.to_string(n)
-      System.get_env(env)
-    end)
-    |> Stream.take_while(& &1)
-    |> Enum.map(&parse_file_system!/1)
+  defp storage() do
+    Application.fetch_env!(:livebook, :storage)
```

Comment:
Add a function on Livebook.Storage module. Either `current()` or `storage()` and import it from there instead. You can use it on `livebook/application.ex` too. The idea is to encapsulate the application env var look in a single place!